.gitignore

```diff
@@ -2,3 +2,4 @@
 /.idea/
 *.pyc
 *.sage.py
+../TAGS
```

Comment:
Should this not just be `TAGS`?  `../` implies the parent dir which is outside of the repo.  Or does that notation mean something else here?

Reply:
You are right - thanks! That explains why the TAGS file was still not being ignored. Now fixed.